### PR TITLE
[DOCS] Fix assumption that 6.7 is last 6.x release

### PR DESCRIPTION
**Pull request by jrodewig**
Relates to elastic/elasticsearch#42255

### CODE REVIEW

File: docs/en/install-upgrade/upgrading-stack.asciidoc [hunk 1]
```diff
@@ -26,14 +26,14 @@ By default, deprecation warnings are logged when the log level is set to `WARN`.
...
-. Upgrade to 6.7 and use the {kibana-ref}/upgrade-assistant.html[{kib} Upgrade Assistant] to {ref}/docs-reindex.html[reindex]
+. Upgrade to 6.8 and use the {kibana-ref}/upgrade-assistant.html[{kib} Upgrade Assistant] to {ref}/docs-reindex.html[reindex]
 any indices that are not compatible with {version}.
 +
 [role="xpack"]
 .Upgrade Assistant
 ******
 The Upgrade Assistant and migration APIs are enabled with both the Basic and
-Trial licenses. You can install the default distribution of 6.7 to use the
+Trial licenses. You can install the default distribution of 6.8 to use the
```
Comment (kat257) [inline docs/en/install-upgrade/upgrading-stack.asciidoc]: Is the deletion of trial licenses relevant here?

Comment (jrodewig) [reply docs/en/install-upgrade/upgrading-stack.asciidoc]: From what I understand, trial licenses can still be used for machine learning features.

File: docs/en/install-upgrade/upgrading-stack.asciidoc [hunk 2]
```diff
@@ -89,11 +89,11 @@ Make sure all 5.x indices have been deleted before upgrading to {version}. {es}
 {version} will fail to start if any 5.x indices are present.
 
 If you are running a version prior to 6.0,
-https://www.elastic.co/guide/en/elastic-stack/6.7/upgrading-elastic-stack.html[upgrade to 6.7]
+https://www.elastic.co/guide/en/elastic-stack/6.8/upgrading-elastic-stack.html[upgrade to 6.8]
 and reindex your old indices or bring up a new {version} cluster and
 {ref}/reindex-upgrade-remote.html[reindex from remote].
 
-The recommended path is to upgrade to 6.7 before upgrading to {version}. This
+The recommended path is to upgrade to 6.8 before upgrading to {version}. This
```
Comment (kat257) [inline docs/en/install-upgrade/upgrading-stack.asciidoc]: Just wonder if this is strong enough language around 'we recommend you upgrade from 6.8 to 7.1' due to changed security licensing. (I admit, I'm having trouble wrapping my brain around the recommended upgrade path.)

Comment (jakelandis) [reply docs/en/install-upgrade/upgrading-stack.asciidoc]: I don't think the changes to license changes the recommendations here. Here it is recommended so you can do a rolling restart (as opposed to a full cluster restart) which was true before and after the license change.

**Comment by jrodewig** [comment]
Backported to 7.x and 7.1.

File: docs/en/install-upgrade/upgrading-stack.asciidoc (hunk 2 above)
Comment (debadair) [reply docs/en/install-upgrade/upgrading-stack.asciidoc]: The recommended upgrade path is through the last minor of the previous major to the most recent minor. So now 6.8 -> 7.1.

**Pull request merged**